doc/adding_collision_detector/adding_collision_checker.rst

```diff
@@ -0,0 +1,17 @@
+Adding a Collision Checker
+==================================
+
+Getting Started
+---------------
```

Comment:
??

Reply:
This will be filled as soon as https://github.com/ros-planning/moveit/pull/1551 is merged.